database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,12 +15,23 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->unsignedBigInteger('level_id')->default(1);
```

Comment:
and also if you are going to update an existing table like this, I suggest to create a new migration for the update. The reason of that is: if other members have executed `php artisan migrate` command on their end, the next time the run the same command, it will not execute the additional changes or update. The reason of that is laravel records every migrations being executed in the `migrations` table and if the migration you updated is already in the record, it will not execute it again. It would be good if there's a changes in your DB tables, please create a new migration.